include/rdma/fi_domain.h

```diff
@@ -167,6 +167,7 @@ struct fi_mr_attr {
 	size_t			auth_key_size;
 	uint8_t			*auth_key;
 	enum fi_hmem_iface	iface;
+	fi_addr_t		addr;
```

Comment:
What is the purpose of this? I don't see it being used

Reply:
dropped.